### PR TITLE
yerp

### PR DESCRIPTION
**Pull request by ghost**
appease the whitespace gods and whatnot

### CODE REVIEW

File: runtime/tutorials/tutor.tutor [hunk 1]
```diff
@@ -207,7 +207,7 @@ is displayed:
 
 Commands to run in the system shell can be highlighted by indenting a line starting with "$".
 
-~~~ sh
+~~~
```
Comment (ghost) [inline runtime/tutorials/tutor.tutor:210]: This is because `vim --version` isn't a shell script, so shouldn't be highlighted as such.

**Comment by fmoralesc** [comment]
@Pyrohh y do u pull request this here instead of helping upstream at fmoralesc/vim-tutor-mode ?!?! The God of PRing to the Correct Repo is not happy!

**Comment by ghost** [comment]
> The God of PRing to the Correct Repo is not happy!

### FUCK

okay PR incoming

**Comment by fmoralesc** [comment]
"language!"

**Comment by ghost** [comment]
Sorry, that was my brother, not me. Damn kids.

**Comment by fmoralesc** [comment]
> Damn kids.

Fucking mingy pests cunts, amirite?

**Comment by ghost** [comment]
> Fucking mingy pests cunts, amirite?

## [ye cunt fukn solid m8](https://www.youtube.com/watch?v=dl6K8ai5J9s&feature=youtu.be)

**Comment by fmoralesc** [comment]
:joy_cat:

**Pull request closed without merging**